packages/atom/test/helpers.ts

```diff
@@ -0,0 +1,60 @@
+import { Context, Operation, main } from 'effection';
+import { performance } from 'perf_hooks';
+
+type World = Context & { spawn<T>(operation: Operation<T>): Promise<T> };
+
+let currentWorld: World;
+
+beforeEach(() => {
+  currentWorld = main(undefined) as World;
+});
+
+afterEach(() => {
+  currentWorld.halt();
+});
+
+export function spawn<T>(operation: Operation<T>): Promise<T> {
+  return currentWorld.spawn(operation);
+}
+
+async function wait(ms: number) {
+  await new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+export async function converge<T>(timeout: number, fn: () => T): Promise<T> {
```

Comment:
My prefered API is to have this just default to the mocha timeout, that way, you don't have to worry about it. Also, what about calling it `when`

Reply:
Yeah, both of those changes seem good to me. I actually just copied this from another bigtest package's test suite; should we make a new package that has this function?

Reply:
Yeah, or maybe just a symlink to a directory in the project root? For now, let's copy-pasta :)

Reply:
I made these changes in place.